src/Raven.Server/Documents/ETL/Providers/Queue/Handlers/Processors/QueueEtlHandlerProcessorForTestAmazonSqsConnection.cs

```diff
@@ -46,7 +46,7 @@ public override async ValueTask ExecuteAsync()
                     context.Write(writer, result);
                 }
             }
-            catch (AmazonSQSException ex) when (ex.ErrorCode == "QueueDoesNotExist")
+            catch (AmazonSQSException ex) when (ex.ErrorCode.Contains("AWS.SimpleQueueService.NonExistentQueue"))
```

Comment:
Do we need full name space here? What about just having `Contains("NonExistentQueue")`? I'm a bit concerned that at some point API will change and the exception will be moved to other location etc

Reply:
fixed